penny_university_frontend/src/actions/index.ts

```diff
@@ -76,10 +79,11 @@ const postFollowUp = (url, followUp) => ({
 })
 
 // These are all thunks. They return a call to dispatch with an action passed into it
-export const loadChatsList = (filter, nextPage) => (dispatch, getState) => {
+export const loadChatsList = (filter: 'all', nextPage?: string) => (dispatch: ThunkDispatch<{}, {}, AnyAction>, getState: () => RootState) => {
```

Comment:
I wonder if we need to curry here and if dispatching a returned action directly would be clearer